lib/middleware/route-action.js

```diff
@@ -0,0 +1,15 @@
+const logger = require('probot/lib/logger');
+/**
+ * Routes the request based on event type
+ */
+module.exports = function route(req, res, next) {
+  const action = req.body.callback_id.substr(0, req.body.callback_id.indexOf('-'));
+
+  if (action) {
+    logger.debug({ ...req.body, token: undefined }, 'Action received from Slack');
+    // Append action derived from callback_id to the request URL to route accordingly
+    req.url += `:${action}`;
```

Comment:
Chose a `:` here to differentiate it from the way `route-events` works. Should we stick with that or use a `.` here just like for `route-events`

Reply:
I don't feel strongly either way, and it's something we can easily change later. I'd like to get rid of this pattern of modifying the routes and go back to an event emitter eventually, but it's not urgent.